src/gluonnlp/embedding/embed_container.py

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=consider-iterating-dictionary, too-many-lines
+"""Load token embedding"""
+
+__all__ = [
+    'StaticEmbedding', 'FastText'
+]
+
+import io
+import logging
+import os
+import warnings
+import fasttext
+import numpy as np
+import mxnet as mx
+from .embed_loader import load_embeddings
+from .evaluation import CosineSimilarity, HyperbolicCosineSimilarity
+from ..data import Vocab
+
+class StaticEmbedding:
```

Comment:
For the AutoML use-cases of data augmentation, we may need more efficient similarity implementations (eg https://github.com/facebookresearch/faiss). So associating a `similarity` function to an embedding class may not be a good way forward. For this PR we can focus on adding the `load_embeddings` and an equivalent for the `fasttext` package and it may not be necessary to add the `embed_container.py`